### PR TITLE
Adding a reverse functionality to sansheets

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: src/sheetFunctions.js [hunk 1]
```diff
@@ -428,3 +428,16 @@ function SAN_TOP_HOLDERS_PERCENT_OF_TOTAL_SUPPLY (projectSlug, from, to, numberO
...
+ * Returns the reversed array of the results
+ * @param {Array} array The array of results
+ * @returns {Array} of reversed results
+ * @customfunction
+ */
+function SAN_REVERSE (array) {
+  [headers, ...rest] = array
+  array.shift()
+
+  return [headers].concat(array.reverse())
```
Comment (IvanIvanoff) [inline src/sheetFunctions.js:442]: We should have a guard for the type and the destructuring is maybe not working as expected (rest does not hold the rest of the array), so something similar to this should work:
```suggestion
  if (!Array.isArray(array) or array.length == 0) {
    return null
  }
 
  headers = array[0]
  array.shift()

  return [headers].concat(array.reverse())
```

File: src/sheetFunctions.js [hunk 2]
```diff
@@ -428,3 +428,22 @@ function SAN_TOP_HOLDERS_PERCENT_OF_TOTAL_SUPPLY (projectSlug, from, to, numberO
...
+ * @returns {Array} of reversed results
+ * @customfunction
+ */
+function SAN_REVERSE (array) {
+  [headers, ...rest] = array
+  array.shift()
+
+  if (!Array.isArray(array) || array.length === 0) {
+    return null
+  }
```
Comment (IvanIvanoff) [inline src/sheetFunctions.js:444]: These checks should be placed before the operations above

Comment (IvanIvanoff) [reply src/sheetFunctions.js:444]: Actually, the `array.shift()` is done twice - above and below. There's some code that needs to be deleted.

Comment (spiderjako) [reply src/sheetFunctions.js:444]: ah yes, I've somehow missed that. Thanks :ok_hand:

**Review by IvanIvanoff — APPROVED** [review]

**Pull request merged**